code/concierge/src/main/kotlin/com/adobe/marketing/mobile/concierge/ui/chat/ConciergeChatViewModel.kt

```diff
@@ -50,6 +52,52 @@ import kotlinx.coroutines.launch
 class ConciergeChatViewModel : AndroidViewModel {
     companion object {
         private const val TAG = "ConciergeChatViewModel"
+        
+        /**
+         * Initializes the welcome config using the parser example
```

Comment:
Can you add a comment here to make a note that this has to be obtained from the service initially